quickstarts/calico/config.yml

```
@@ -0,0 +1,48 @@
+id:
+slug: calico newrelic
+title: Calico Newrelic
+
+description: |
+
+  Calico is an open source networking and network security solution for containers, virtual machines, and native host-based workloads. Calico supports a broad range of platforms including Kubernetes, OpenShift, Mirantis Kubernetes Engine (MKE), OpenStack, and bare metal services.
+
+Whether you opt to use Calico's eBPF data plane or Linux’s standard networking pipeline, Calico delivers blazing fast performance with true cloud-native scalability
```

I think this line is breaking the yaml formatting, it needs to be on the same indentation as the rest of the text block.
```suggestion
  Whether you opt to use Calico's eBPF data plane or Linux’s standard networking pipeline, Calico delivers blazing fast performance with true cloud-native scalability
```

---

Updated